### DISCUSSION TITLE
randomly says "Stopping parent process [num]"

### DISCUSSION THREAD

**Discussion by jmugan** [Kludex/uvicorn#1635] [Q&A]
I'm having an issue in production where uvicorn just randomly says `Stopping parent process [num]` and terminates. Maybe it is running out of memory? It looks like it has plenty. Does anyone know where I can begin to track this down or become robust to it? I don't have deep knowledge of OS or network stuff. Thanks!

**Comment by iudeen** [comment]
Can you give bit more information on how you're running the server?

Are you using workers?
Are you using a process manager (Eg: Gunicorn)?

**Reply by jmugan** [reply]
Thanks for your reply. I'm running it like this
```
if __name__ == "__main__":
    import uvicorn
    uvicorn.run("src.app.main:app", host="0.0.0.0", port=8000, log_level="info",
                workers=4)
```

**Reply by iudeen** [reply]
What version of uvicorn are you running?
And, is there a way we can reproduce the issue?

**Reply by jmugan** [reply]
I'm running "Running uvicorn 0.15.0 with CPython 3.8.8 on Linux." I can't reproduce the issue, it just happens randomly and I learn about it through a slack message :)

**Reply by iudeen** [reply]
I tried multiple ways to reproduce, I am unable to :(. And at work we are running uvicorn in production for over an year now, and we did not face this issue.

Can you use latest version of Uvicorn and see that fixes this?

**Reply by jmugan** [reply]
Yep, I'll try that, thanks!
